src/shared/seccomp-util.h

```diff
@@ -38,11 +38,11 @@ typedef struct SyscallFilterSet {
 } SyscallFilterSet;
 
 enum {
+        SYSCALL_FILTER_SET_DEFAULT,
```

Comment:
maybe add a comment after this one saying: "please leave 'default' first, but sort the rest alphabetically, thanks" or so.